torch/onnx/_internal/onnx_proto_utils.py

```diff
@@ -287,3 +291,25 @@ def _find_onnxscript_op(
                 else None,
             )
     return onnx_function_list, included_node_func
+
+
+def _convert_tensor_to_numpy(input: Any) -> Any:
+    if isinstance(input, torch.Tensor):
```

Comment:
Moving numpy import to here will resolve CI issue. The rule is not to make `import torch` depend on `numpy`.

Reply:
Thanks!